packages/devextreme/testing/tests/DevExpress.ui.widgets/stepper.tests.js

```diff
@@ -62,6 +62,87 @@ QUnit.module('Initialization', moduleConfig, () => {
     });
 });
 
+QUnit.module('Navigation', moduleConfig, () => {
+    QUnit.test('In linear mode only next or previous steps can be selected on click', function(assert) {
+        this.reinit({
+            items: [{}, {}, {}, {}],
+            selectedIndex: 1,
+            linear: true,
+        });
+
+        this.getItems().eq(3).trigger('dxclick');
+
+        assert.equal(this.instance.option('selectedIndex'), 1, 'selectedIndex not changed');
+
+        this.getItems().eq(2).trigger('dxclick');
+
+        assert.equal(this.instance.option('selectedIndex'), 2, 'selectedIndex changed');
+    });
+
+    [true, false].forEach((linear) => QUnit.test(`selectionChanged callback should not be triggered when is already selected, linear=${linear}`, function(assert) {
```

Comment:
Could you please update formatting to stop this function return result of ` QUnit.test`, how we do it in another places? It's more readable for me
```suggestion
    [true, false].forEach((linear) => {
        QUnit.test(`selectionChanged callback should not be triggered when is already selected, linear=${linear}`, function(assert) {
```

Reply:
fixed